components/movementsensor/replay/replay_test.go

```diff
@@ -183,24 +195,28 @@ func TestNewReplayMovementSensor(t *testing.T) {
 func TestReplayMovementSensorFunctions(t *testing.T) {
```

Comment:
I would separate the property tests into a separate test function with its own cases/tests as the properties tests are just focused on if any data is present that these tests focus on if the exact amount of data is available based on the filter